doc/releases/changelog-dev.md

```diff
@@ -0,0 +1,779 @@
+
+<h3>Improvements 🛠</h3>
+
+<h4>A more flexible projector</h4>
+
+* `qml.Projector` now accepts a state vector representation, which enables the creation of projectors
+  in any basis.
+  [(#4192)](https://github.com/PennyLaneAI/pennylane/pull/4192)
+
+  ```python
+  dev = qml.device("default.qubit", wires=2)
+  @qml.qnode(dev)
+  def circuit(state):
+      return qml.expval(qml.Projector(state, wires=[0, 1]))
+  zero_state = [0, 0]
+  plusplus_state = np.array([1, 1, 1, 1]) / 2
+  ```
+
+  ```pycon
+  >>> circuit(zero_state)
+  1.
+  >>> 
+  >>> circuit(plusplus_state)
+  0.25
+  ```
+
+<h4>Do more with qutrits</h4>
+
+* Three qutrit rotation operators have been added that are analogous to `RX`, `RY`, and `RZ`:
+
+  - `qml.TRX`: an X rotation
+  - `qml.TRY`: a Y rotation
+  - `qml.TRZ`: a Z rotation
+
+  [(#2845)](https://github.com/PennyLaneAI/pennylane/pull/2845)
+  [(#2846)](https://github.com/PennyLaneAI/pennylane/pull/2846)
+  [(#2847)](https://github.com/PennyLaneAI/pennylane/pull/2847)
+  
+* Qutrit devices now support parameter-shift differentiation.
+  [(#2845)](https://github.com/PennyLaneAI/pennylane/pull/2845)
+
+<h4>The qchem module</h4>
+
+* `qchem.molecular_hamiltonian()`, `qchem.qubit_observable()`, `qchem.import_operator()`, and `qchem.dipole_moment()` now return an arithmetic operator if `enable_new_opmath()` is active.
+  [(#4138)](https://github.com/PennyLaneAI/pennylane/pull/4138)
+  [(#4159)](https://github.com/PennyLaneAI/pennylane/pull/4159)
+  [(#4189)](https://github.com/PennyLaneAI/pennylane/pull/4189)
+  [(#4204)](https://github.com/PennyLaneAI/pennylane/pull/4204)
+
+* Non-cubic lattice support for all electron resource estimation has been added.
+  [(3956)](https://github.com/PennyLaneAI/pennylane/pull/3956)
+
+* The `qchem.molecular_hamiltonian()` function has been upgraded to support custom wires for constructing
+  differentiable Hamiltonians. The zero imaginary component of the Hamiltonian coefficients have been
+  removed.
+  [(#4050)](https://github.com/PennyLaneAI/pennylane/pull/4050)
+  [(#4094)](https://github.com/PennyLaneAI/pennylane/pull/4094)
+
+* Jordan-Wigner transforms that cache Pauli gate objects have been accelerated.
+  [(#4046)](https://github.com/PennyLaneAI/pennylane/pull/4046)
+
+* An error is now raised by `qchem.molecular_hamiltonian` when the `dhf` method is used for an 
+  open-shell system. This duplicates a similar error in `qchem.Molecule` but makes it clear
+  that the `pyscf` backend can be used for open-shell calculations.
+  [(#4058)](https://github.com/PennyLaneAI/pennylane/pull/4058)
+
+<h4>Next-generation device API</h4>
+
+* The new device interface has been integrated with `qml.execute` for autograd, backpropagation, and no differentiation.
+  [(#3903)](https://github.com/PennyLaneAI/pennylane/pull/3903)
+
+* Support for adjoint differentiation has been added to the `DefaultQubit2` device.
```

Comment:
Is this the first time we're mentioning default qubit 2? 🤔 Little confused here

Reply:
We discussed offline :+1: